### DISCUSSION TITLE
Empty expression on x-intersect throws error

### DISCUSSION THREAD

**Discussion by james0r** [alpinejs/alpine#3640] [7. Bugs (archived)]
for

```javascript
x-intersect=""
```
and 
```javascript
x-intersect
```

I'm getting
```
Uncaught SyntaxError: Unexpected token '}'
```

Would be nice if this soft-failed like expressions do in Alpine core directives.

**Comment by ekwoka** [comment]
probably better than soft failing would be to just explicitly call out that the directive requires an expression.

**Reply by james0r** [reply]
That would be better for sure, was just thinking for consistency with alpine core.

**Reply by ekwoka** [reply]
Could probably be implemented in the evaluator directly.
